build_on_raijin.sh fails: can't find NetCDF

Here's what happens when I try to compile:
```
$ module list
Currently Loaded Modulefiles:
  1) pbs            2) dot            3) ncview/2.1.2   4) git/2.9.5
$ cd /short/v45/aek156/sources/new/libaccessom2/
$ ./build_on_raijin.sh
mpifort executable found:
Will assume system MPI implementation is sound. Remove mpifort from PATH to automatically configure MPI
-- Failed to find NetCDF interface for F90
CMake Error at /apps/CMake/3.6.2/share/cmake-3.6/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find NetCDF (missing: NETCDF_LIBRARIES NETCDF_INCLUDE_DIRS
  NETCDF_HAS_INTERFACES)
Call Stack (most recent call first):
  /apps/CMake/3.6.2/share/cmake-3.6/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindNetCDF.cmake:119 (find_package_handle_standard_args)
  CMakeLists.txt:38 (find_package)


-- Configuring incomplete, errors occurred!
See also "/short/v45/aek156/sources/new/libaccessom2/build/CMakeFiles/CMakeOutput.log".
/short/v45/aek156/sources/new/libaccessom2
```

--- a/util/make_dir/config.nci
+++ b/util/make_dir/config.nci
@@ -1,5 +1,5 @@
 module purge
 module load intel-fc/17.0.1.132
 module load intel-cc/17.0.1.132
-module load netcdf
+module load netcdf/4.3.2
 module load openmpi/1.10.2

